gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/DagProc.java

```diff
@@ -75,8 +91,19 @@ public final void process(DagManagementStateStore dagManagementStateStore,
       dagProcEngineMetrics.markDagActionsInitialize(getDagActionType(), false);
       throw e;
     }
-    act(dagManagementStateStore, state, dagProcEngineMetrics);
-    log.info("{} concluded processing for dagId : {}", getClass().getSimpleName(), this.dagId);
+    try {
+      act(dagManagementStateStore, state, dagProcEngineMetrics);
+    } catch (Exception e) {
+      if (isTransientException(e)) {
```

Comment:
isn't this the reverse of what we want? we _should_ retry transient exceptions not NON-transient?

Reply:
good catch, updating